Swift_Alert/Swift_Alert/Extension/Extension + String.swift

```diff
@@ -45,7 +46,26 @@ extension String {
         return result
     }
 
+    func formatCardNumber(with mask: String) -> String {
```

Comment:
```suggestion
    func formatCardNumber(with mask: String) -> String {
```
это же логика ее в vm